lib/fabrik/Fabrik.js

```diff
@@ -41,11 +42,20 @@ class Fabrik {
         const instance = manager.createInstance(instance_id);
         return Promise
           .try(() => context ? context : instance.platformContext)
-          .then(context => instance.assignPlatformManager(BasePlatformManager.getInstance(instance_id, context)))
+          .then(context => instance.assignPlatformManager(Fabrik.getPlatformManager(context.platform)))
           .return(instance);
       });
   }
 
+  static getPlatformManager(platform) {
+    const PlatformManager = (platform && CONST.PLATFORM_MANAGER[platform]) ? require(`./${CONST.PLATFORM_MANAGER[platform]}`) : undefined;
```

Comment:
attribute name should start with small case here - `platformManager`

Reply:
As it is a class reference, kept it as `PlatformManager`